### ISSUE TITLE
Publish on Artifact Hub

### ISSUE THREAD

**Issue by bbkz** [wger-project/helm-charts#26]
Publish the chart on the Artifact Hub:

* https://artifacthub.io/docs/topics/repositories/

**Comment by bbkz** [comment]
I created a new organization `wger-project` on https://artifacthub.io and added the repository to it. What i basically did:
 
* Add a new account
* Add a wger-project organization
* Switch to the context of the organization
* Add a new repository and copy the ID
* Got to the gh-pages branch and add the `artifacthub-repo.yml` file, including the ID

I did not yet add the owner and made it the official one. @rolandgeider Maybe you wan't to create a account so that i can add you to the organization and add you as owner...

* https://artifacthub.io/packages/helm/wger/wger

**Comment by rolandgeider** [comment]
cool, thanks! (also for keeping on maintaining this 😄)

I made an account on artifacthub, "rge"

**Comment by rolandgeider** [comment]
BTW, is https://wger-project.github.io/helm-charts/ still the correct URL or did something break? https://github.com/wger-project/helm-charts/deployments/github-pages

**Comment by bbkz** [comment]
Of course, i'm happy to do that. I'll then try to get the "offical" tag on artifactshub.io

https://wger-project.github.io/helm-charts/ is still the valid URL for the helm repository, on the gh-page branch was a old copy of the source code with a README.md, which was displayed when accessing trough the browser, i removed that.

I am not sure, but maybe i'll add a README.md linking to the new README.md location... 🤔 

For the helm repository the `index.yaml` is relevant.

**Comment by bbkz** [comment]
The offical badge has been added:
 
* https://github.com/artifacthub/hub/issues/4268#event-16158015095

🎉

**Comment by rolandgeider** [comment]
🥳🥳
